layers/meta-resin-raspberrypi/recipes-core/images/resin-image.bbappend

```diff
@@ -36,3 +36,5 @@ python overlay_dtbs_handler () {
 
 addhandler overlay_dtbs_handler
 overlay_dtbs_handler[eventmask] = "bb.event.RecipePreFinalise"
+
+IMAGE_INSTALL_append_rpi=" enable-overcommit"
```

Comment:
The usual style is to add spaces around the equals sign.

Reply:
Done. thank you :)